src/pages/dashboard/gig-board-create/settings.tsx

```diff
@@ -0,0 +1,92 @@
+import { Field, Form, Formik } from 'formik';
+import React, { useContext, useState } from 'react';
+import { SkillsInput } from '../../../components/Form/skills-input';
+import GigBoard from '../../../components/GigBoard';
+import StarterKitContext from '../../../context/starterKit';
+import { ServiceStatusEnum } from '../../../types';
+
+const BASE_URL = 'http://localhost:3000';
```

Comment:
Not really a good practice to have that here. 
- Solution 1: get the base URL dynamically with window.location
- Solution 2: put this in a .env
Sol 1 seems good to me here